test/check-application

```diff
@@ -209,14 +209,6 @@ class TestFiles(testlib.MachineCase):
         b.set_input_text("input[placeholder='Filter directory']", "")
         self.browser.wait_js_cond("ph_count('#folder-view tbody tr') > 1")
 
-        # Big file downloads fine, runs in this test as running two downloads
-        # in the same test causes issues with Chromium. This takes ~ 40 seconds.
-        m.execute("truncate -s 1500M /home/admin/test.iso")
```

Comment:
Right, very confused why we did this back then, can't really find the reference.